content/_releases/24/45.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Product Update - v24.45
+description: Details of the product update v24.45 from the 7nd of November 2024.
+layout: rel
+version: "24.45"
+section: Product Updates
+category: product-updates
+releaseDate: 2024-11-07
+redirect_from:
+  - /releases/24.45.html
+---
+
+## New Features
+### Editable delay for container shutdown in Ordinary Flows
+Introducing the ability to adjust the container shutdown delay for Ordinary Flows, enhancing user control and flexibility within the system.
+By introducing the `shut_down_timeout_enabled` feature flag at the Tenant level, users can control the delay duration for container shutdown on a per-flow basis. The addition of the shut_down_timeout attribute within Flows allows users to define specific shutdown timeframes (0 to 30 minutes) to better suit their workflow requirements.
+With restrictions in place for long-running (real-time) flows and a dedicated `Shutdown` section in the settings menu of the flow designer (based on the feature flag status), users can efficiently manage container shutdown times, enhancing operational control and system performance.
+
+_Note: The shutdown_timeout does not take effect immediately after updating the flow. It can take up to 10 minutes (usually 3-4 minutes) before the Kubernetes pods exhibit an increased drop time._
+
+{% include img.html max-width="100%" url="/assets/img/RN/2445/Shutdown.png" title="Shutdown" %}
+
+
+
+### Extending `SCHEDULED_EXECUTION_TIMEOUT` to 720 seconds
+Enhancing the system's timeout configuration by increasing the `SCHEDULED_EXECUTION_TIMEOUT` component environment variable's maximum value to 720 seconds. This adjustment caters to systems that necessitate more time for retrieving select model and dynamic metadata during one-time executions. By elevating the timeout threshold from the previous limit of 360 seconds, this update enables smoother execution of processes that demand longer data retrieval periods, ensuring optimal performance and responsiveness for time-critical operations within the system.
+
+### Introducing `allow_modification` attribute for Recipe editing
```

Comment:
not recipe editing, flow editing

Introducing `allow_modification` attribute for Recipe